views/templates/block/activityLog.twig

```diff
@@ -19,6 +19,36 @@
             <div id="errorDuringUpgrade" class="col-xs-12" style="display:none;">
                 <h4>{{ 'Errors'|trans({}, 'Modules.Autoupgrade.Admin') }}</h4>
                 <div id="infoError" class="processing"></div>
+                <button id="reportErrorButton" class="btn btn-warning" style="margin-top: 10px;" data-toggle="modal" data-target="#errorModal">
+                    {{ 'Report errors'|trans({}, 'Modules.Autoupgrade.Admin') }}
+                </button>
+            </div>
+
+            <div id="errorModal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+                <div class="modal-dialog modal-dialog-centered" role="document">
+                    <div class="modal-content">
+                        <div class="modal-header" style="display: flex; justify-content: space-between; align-items: center;">
```

Comment:
Was the default display broken?

Reply:
After looking in detail, you need to put the close button before the title